workflow/Activity/CalculateBill.go

```diff
@@ -0,0 +1,14 @@
+package activity
+
+import (
+	model "github.com/E-Furqan/Food-Delivery-System/Models"
+)
+
+func (act *Activity) CalculateBill(CombineOrderItem model.CombineOrderItem, items []model.Items) (float64, error) {
```

Comment:
we can keep this with other order activities.

Reply:
i have moved it  order activity file